pkg/cmd/cli/backup/delete.go

```diff
@@ -22,9 +22,11 @@ import (
 
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
I would use camelCase: `apierrors` -> `apiErrors`

Reply:
@mateusoliveira43  For package names (and package name aliases) the general convention is all lowercase